### PR TITLE
Country Picker for Vote Results + cleanup

### PR DESCRIPTION
**Pull request by t27duck**
* When viewing a poll, you may select a country to see votes from only
  that country
* Moved various functions and logic into separate functions in separate
  files
* Implemented all CSS into a separate file. Removed the CMOC css
* Added the RC24 logo to the little vote card things in the banner
  graphic because why not
* Made links a bit more stylish
* Handle case where a poll with no estimation votes would result in a
  division by zero

TLDR - Looks like this

Screen Shot 2020-01-04 at 1 36 56 PM

### CODE REVIEW

**Comment by Rexogamer** [comment]
question: why remove CMOC CSS?

**Comment by t27duck** [comment]
> question: why remove CMOC CSS?

To keep things separate. Both pages serve different purposes and it would be relatively easy to get both styles conflicting. So instead, I opted to take parts of the other CSS file and integrate it into this page's styles.

Also, having the CMOC background as a border kinda clashed.

**Comment by KcrPL** [comment]
Looks hella better ^^
